GeoEntity.md

```diff
@@ -0,0 +1,281 @@
+    
+	- Example URL: https://api.careerbuilder.com/core/geography/entities/admin_area_2s?country=us&admin_area_one=ga
+	
+- Response Sample:
+```
+{
+  "data": {
+    "entities": [
+      {
+        "long_name": "Camden County",
+        "short_name": "Camden County",
+        "level": 2,
+        "place_id": "ChIJl6NPkwkb5YgRwGvHFXP6tYY",
+        "location": {
+          "lat": 30.898276,
+          "lng": -81.6035062
+        },
+	"viewport": {
+          "northeast": {
+            "lat": 31.169595,
+            "lng": -81.40354649999999
+          },
+          "southwest": {
+            "lat": 30.7114626,
+            "lng": -81.936988
+          }
+        },
+        "formatted_address": "Camden County, GA, USA"
+      },
+      ...
+  ]
+ }
+}
+```
+&nbsp;
+----------------------------
+#All Localities Request
+- Request Structure:
+	- Target: https://api.careerbuilder.com/core/geography/entities/localities
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    | admin_area_one | Optional  | A first order administrative subdivision, both long name and short name are accepted. If present, limits the result set to the localities within the specified first-level administrative area. Otherwise, all localities within the specified country will be returned.|
+    
+	- Example URL: https://api.careerbuilder.com/core/geography/entities/localities?country=us&admin_area_one=ga
+	
+- Response Sample:
+```
+{
+  "data": {
+    "entities": [
+      {
+        "locality": "Candler-McAfee",
+        "place_id": "ChIJ9T6vBZ4A9YgRMKirj_zfz3I",
+        "location": {
+          "lat": 33.7254622,
+          "lng": -84.2752831
+        },
+        "viewport": {
+          "northeast": {
+            "lat": 33.7399031,
+            "lng": -84.230871
+          },
+          "southwest": {
+            "lat": 33.7122249,
+            "lng": -84.30969189999999
+          }
+        },
+        "formatted_address": "Candler-McAfee, GA 30032, USA"
+      },
+      ...
+    ]
+  }
+}
+```
+&nbsp;
+----------------------------
+#All Postal Codes Request
+- Request Structure:
+	- Target: https://api.careerbuilder.com/core/geography/entities/postal_codes
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    | admin_area_one | Optional  | A first order administrative subdivision, both long name and short name are accepted. If present, limits the result set to the postal codes ithin the specified first-level administrative area. Otherwise, all postal codes within the specified country will be returned.|
```

Comment:
Sorry, I still don't get it yet.
So you recommend me to change from "within the specified" to "within"?
or you mean I have lacked a letter "w" in word "within" (I don't see `w` missing)?